### PR TITLE
feat: added screens

### PR DESCRIPTION
**Pull request by msdeshmukh009**
### Pages Created 
- Authentication
- Single video page
- Single playlist page

### Preview
screens-3

### CODE REVIEW

File: src/components/video-card/VideoCard.jsx [hunk 1]
```diff
@@ -1,54 +1,40 @@
...
-      </button>
-
-      <div className="video-card-image-container">
+    <div className="video-card">
+      <Link to={`/explore/${_id}`} className="video-card-image-container">
         <img
           className="responsive-img"
-          src="https://i.ytimg.com/vi/iMVgvkVJuDI/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLBvU99zlZVetW12JDJFVPTRzhKlAg"
-          alt="thumbnail"
+          src={`https://i.ytimg.com/vi/${_id}/hq720.jpg`}
```
Comment (gopalbbharadva) [inline src/components/video-card/VideoCard.jsx]: Suggestion: you can make helper function , which takes id and returns the imgSrc, so your src attribute looks cleaner.

**Review by gopalbbharadva — APPROVED** [review]
Code looks cleaner. UI is also good.

**Review by manojkumar8400 — APPROVED** [review]
good to go.

**Review by bipin7yadav — APPROVED** [review]
ui looks great

File: src/components/video-card/VideoCard.jsx (hunk 1 above)
Comment (msdeshmukh009) [reply src/components/video-card/VideoCard.jsx]: Will do that thanks!!

**Pull request merged**